tests/hooks/test_aap_hook_migration.py

```
@@ -0,0 +1,233 @@
+    disk transfer and PostHook ran after VM creation.
+    """
+
+    storage_map: StorageMap
+    network_map: NetworkMap
+    plan_resource: Plan
+
+    def test_create_storagemap(
+        self,
+        prepared_plan: dict[str, Any],
+        fixture_store: dict[str, Any],
+        ocp_admin_client: "DynamicClient",
```

Ruff (0.15.20)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)